src/sv-pipeline/05_annotation/scripts/compute_AFs.py

```diff
@@ -360,51 +348,54 @@ def main():
         '##INFO=<ID=FREQ_HOMALT,Number=1,Type=Float,Description="Homozygous alternate genotype frequency (biallelic sites only).">',
         '##INFO=<ID=CN_NUMBER,Number=1,Type=Integer,Description="Total number of samples with estimated copy numbers (multiallelic CNVs only).">',
         '##INFO=<ID=CN_COUNT,Number=.,Type=Integer,Description="Number of samples observed at each copy state, starting from CN=0 (multiallelic CNVs only).">',
+        '##INFO=<ID=CN_STATUS,Number=.,Type=Integer,Description="Differnt copy states observed, starting from CN=0 (multiallelic CNVs only).">',
```

Comment:
```suggestion
        '##INFO=<ID=CN_STATUS,Number=.,Type=Integer,Description="Different copy states observed, starting from CN=0 (multiallelic CNVs only).">',
```
What is CN_STATUS and why are we adding it? It looks like it's just 0,1,...,maxCN, is that right? But we may not observe every copy state between 0 and maxCN so this description does not seem accurate. What is it used for?